lib/convert.js

```diff
@@ -34,14 +50,16 @@ const opt_string = (value) => {
 
 const opt_tag = (tag) => {
   const retrived_tag = {};
-  const pairs = tag.split(',').map((pair) => pair.split('='));
+  const tag_arr = tag.match(/(\s*([^,]+)".*?"|\s*([^,]+)[^",\s]+)(?=\s*,|\s*$)/g); //same regex as in param_arr
 
-  pairs.forEach((pair) => {
-    if (pair[0] === 'Key') {
-      retrived_tag.Key = pair[1];
+  tag_arr.forEach((tag) => {
+    const split_tag = tag.split(/=(.+)?/);
+
+    if (split_tag[0] === 'Key') {
+      retrived_tag.Key = split_tag[1];
     } 
-    else if (pair[0] === 'Value') {
-      retrived_tag.Value = pair[1];
+    else if (split_tag[0] === 'Value') {
```

Comment:
Is there a chance these could be the same? Would running the `replace` on a key be fatal? In which case you can just do (using same variables as my reduce above)

```
    if (key === 'Key' || key === 'Value') {
      retrieved[key] = value.replace(/^"(.*)"$/, '$1');
    } else {
      throw new Error('Missing Key or Value inside tag');
    }
```